packages/prettier-plugin-java/test/hello-world/_output.java

```diff
@@ -1,5 +1,46 @@
 package graphics.x.y.z;
-public class HelloWorld <AR> extends APZnd implements Leoq, LA {
-  public int t = expression;
-  public void test( int maz, final unannClassType go) throws testException block
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+public class HelloWorld <AR, AKDa> extends APZnd implements Leoq, LA {
```

Comment:
We will need to split this to **multiple** smaller test cases

Reply:
How do you think we should proceed ? One test for each printer function ?